### PR TITLE
[RAPTOR-6477] make uwsgi an extra dependency for DRUM

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
I changed `uwsgi` to be an extra dependency for `mlpiper` instead of mandatory. Thus also making it an extra dependency for DRUM.

+ Fix docs

### CODE REVIEW

**Comment by yakov-g** [comment]
jarvis test functional_training_model_templates

File: docker/java_dropin_env_base/Dockerfile [hunk 1]
```diff
@@ -38,5 +38,6 @@ RUN pip3 install --no-cache-dir setuptools wheel
 
 
 # do the trick to install drum deps
-RUN pip3 install --no-cache-dir datarobot-drum
+# uwsgi is an extra dependency and it is not installed together with DRUM, so install it explicitly
+RUN pip3 install --no-cache-dir datarobot-drum uwsgi
```
Comment (meltzg) [inline docker/java_dropin_env_base/Dockerfile:42]: we don't actually use it though in our environments. can we skip it?

**Review by meltzg — APPROVED** [review]
would like it removed from the env base images unless we really need it. Otherwise g2g

**Comment by yakov-g** [comment]
> would like it removed from the env base images unless we really need it. Otherwise g2g
Currently it is in the base images anyway, as previously mlpiper installed uwsgi.

**Review by drenfr01 — APPROVED** [review]
LGTM - but I assume we don't need to update the environments because this is only a local change?

**Pull request merged**